packages/vscode-extension/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "rootDir": ".",
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "strict": true /* enable all strict type-checking options */,
+    "strict": false /* enable all strict type-checking options */,
```

Comment:
Why is `strict` disabled?

Reply:
axios 1.7.4 need typescript 4.7+, which will cause too many compile errors : try {...} catch(e){...}

Reply:
Please try to use `"useUnknownInCatchVariables": false` flag.